src/data/polynomial/div.lean

```diff
@@ -430,6 +431,20 @@ end
 
 variable {R}
 
+lemma eval_ring_hom_ker (x : R) : (eval_ring_hom x).ker = ideal.span {X - C x} :=
+by { ext y, simpa only [ideal.mem_span_singleton, dvd_iff_is_root] }
+
+lemma aeval_ker (x : R) : ((aeval x : R[X] →ₐ[R] R) : R[X] →+* R).ker = ideal.span {X - C x} :=
+eval_ring_hom_ker x
```

Comment:
Instead of this lemma, we should havea lemma that says `((aeval x : R[X] →ₐ[R] R) : R[X] →+* R) = (eval_ring_hom x)`

Reply:
Without this (or the `ker` lemma), the proof of `quotient_span_X_sub_C_alg_equiv` looks slightly uglier. We can certainly replace the explicit type ascriptions with `rw` or `simp` but that just makes it even longer.